### ISSUE TITLE
FunctionClauseError: reply/2

### ISSUE THREAD

**Issue by woylie** [livehelpnow/tds#51]
I'm getting a function clause error when the tcp connection fails.

```
15:00:28.700 [error] GenServer #PID<0.8450.0> terminating
** (FunctionClauseError) no function clause matching in Tds.Utils.reply/2
    (tds) lib/tds/utils.ex:50: Tds.Utils.reply(%Tds.Error{message: "tcp connect: nxdomain", mssql: nil}, %Tds.Protocol{env: %{trans: <<0>>}, itcp: nil, opts: [<redacted>], pak_data: "", pak_header: "", query: nil, result: nil, sock: nil, state: :ready, tail: "", transaction: nil, usock: nil})
    (tds) lib/tds/utils.ex:46: Tds.Utils.error/2
    (db_connection) lib/db_connection/connection.ex:134: DBConnection.Connection.connect/2
    (connection) lib/connection.ex:622: Connection.enter_connect/5
    (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
Last message: nil
State: Tds.Protocol
```

**Comment by mjaric** [comment]
That is obsolete code which need to be replaced with proper error reporting to DBConnection module function. It won't cause any issues, but it is annoying to see it in logs. I will do fix in next release

**Comment by mjaric** [comment]
fixed int v1.0.9
